indico/modules/users/templates/users_admin.html

```diff
@@ -79,10 +80,15 @@ <h3>
                                             {{ user_entry.full_name }}
                                         {%- endif -%}
                                     </span>
+                                    {% if user_entry.user %}
+                                        {{ labels(user_entry.user) }}
+                                    {% endif %}
                                     <span class="right">
                                         {% if user_entry.user and user_entry.user.is_system %}
                                             <em title="{% trans %}This is the system user.{% endtrans %}">
```

Comment:
I don't think we need to keep the `<em>` here; the title could be set directly on the span below